staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -470,6 +471,7 @@ type DeleteOptions struct {
 
 	// Must be fulfilled before a deletion is carried out. If not possible, a 409 Conflict status will be
 	// returned.
+	// +k8s:conversion-gen=false
```

Comment:
worth a comment why

Reply:
done

Reply:
Maybe sth like:
"Converting url.Values to Preconditions requires manual conversion."

Reply:
> Maybe sth like:
> "Converting url.Values to Preconditions requires manual conversion."

addressed.